test/selfSpecs/useExistingProject.test.ts

```diff
@@ -1,20 +0,0 @@
-/*
- * Copyright (c) 2024, salesforce.com, inc.
- * All rights reserved.
- * Licensed under the BSD 3-Clause license.
- * For full license text, see LICENSE.txt file in the repo root or https://opensource.org/licenses/BSD-3-Clause
- */
-
-import { step } from 'mocha-steps';
-import { TestSetup } from '../testSetup.ts';
-import * as utilities from '../utilities/index.ts';
-
-describe('Use existing project', async () => {
-  let testSetup: TestSetup;
-  step('verify existing project is open', async () => {
-    testSetup = new TestSetup('UseExisProject');
```

Comment:
Isn't this the old way? Is it supposed to be like that?

Reply:
This was deleted by me. Since I do not think we have used it anywhere.

Reply:
oh I missed that haha, sorry!